packages/fast-components-react-base/src/toggle/toggle.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+
+import {IToggleProps} from "./toggle.props";
+import {IToggleClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/**
+ * Toggle state interface
+ */
+export interface IToggleState {
+    checked: boolean;
+}
+/**
+ * Toggle base component
+ */
+/* tslint:disable-next-line */
```

Comment:
I think we can remove this tslint rule

Reply:
ok - it's in the base component you put up so it should be removed from there too

Reply:
for sure